docs/user/kotlin-impl-spec-comparison.md

```diff
@@ -0,0 +1,62 @@
+# `partiql-lang-kotlin` compared to the PartiQL Specification
+
+This document outlines the ways in which the `partiql-lang-kotlin` reference implementation conforms and diverges from
+the [PartiQL specification](https://partiql.org/assets/PartiQL-Specification.pdf).
+Also included are some features that are part of the Kotlin implementation but are not yet formally defined in the 
+PartiQL spec.
+
+Note: the list is only partially completed. Sections 6.2 onwards from the spec have yet to be compared with the Kotlin
+implementation.
+
+## Features defined in spec
+
+| Spec section	              |Feature	|Implemented in `partiql-lang-kotlin`	| Notes	                                                                                                                                                                                                             |
+|----------------------------|---	|---	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 3.3	                       |Syntax improvement `SELECT` optionally written as last clause	|No	| 	                                                                                                                                                                                                                  |
```

Comment:
Improvement and not a blocker: we could have direct links to the Spec. like:

```suggestion
| [3.3](https://partiql.org/assets/PartiQL-Specification.pdf#subsection.3.3)	                       |Syntax improvement `SELECT` optionally written as last clause	|No	| 	                                                                                                                                                                                                                  |
```

Reply:
Will do. TIL you could link to specific sections of a PDF